src/openms/cmake_findExternalLibs.cmake

```diff
@@ -57,7 +57,7 @@ find_package(XercesC REQUIRED)
 
 #------------------------------------------------------------------------------
 # BOOST
-find_boost(iostreams date_time math_c99 regex)
+find_boost(iostreams date_time math_c99 regex system thread)
```

Comment:
this will link to additional boost libraries, I hope we are building those in contrib?

Reply:
seems like we are not doing that for the contrib and for any of the test systems. @jpfeuffer  @timosachsenberg  any comments on how to handle this? should we use `std::mutex` instead of the more fine grained locking?

Reply:
I would go with boost here. c11 and even c14 is lacking too many basic features. It’s afaik also much more stable.

Reply:
I agree with Timo. And since we include boost already anyhow and it seems to be the fastest one, let's go with boost. Should be easy to add to the contrib as configure flags.